cdap-master/src/main/java/co/cask/cdap/master/startup/ConfigurationCheck.java

```diff
@@ -125,4 +137,14 @@ private boolean isPositiveInteger(String key) {
       return false;
     }
   }
+
+  private boolean isValidKafkaTopic(String kafkaTopic) {
+    try {
+      Topic.validate(kafkaTopic);
+    } catch (Exception e) {
+      LOG.error("  {} is not a valid kafka topic", kafkaTopic);
```

Comment:
nit: the (Topic.validate)[https://github.com/apache/kafka/blob/trunk/core/src/main/scala/kafka/common/Topic.scala#L30] method has descriptive error messages. We should log the messages too.

Also, perhaps only catch ``InvalidTopicException`` here?